Language/Variables/Constants/floatingPointConstants.adoc

```diff
@@ -34,7 +34,7 @@ Similar to integer constants, floating point constants are used to make code mor
 
 [source,arduino]
 ----
-n = 0.005;  // 0.005 is a floating point constant
+const float n = 0.005;  // 0.005 is a floating point constant
```

Comment:
```suggestion
float n = 0.005;  // 0.005 is a floating point constant
```
Even though "`const` correctness" is very much best practices, the use of `const` here isn't helpful for the code snippet's purpose of demonstrating the use of a "floating point constant". We don't know what the programmer's intention for `n` might be, so it's not sure whether the use of `const` would be best practices, or result in a compilation error from an intentional assignment to `n` later in the program. Meanwhile, the beginner might think the "constant" in "floating point constant" is related to the use of `const`. Many beginners will not be familiar with `const` because the Arduino example code eschews `const` correctness.

Reply:
I agree with you.

Reply:
This is the first time for me to contribute to arduino, I just wonder when could I see the effect of this commit in arduino reference site ?

Reply:
>This is the first time for me to contribute to arduino

Welcome!
>I just wonder when could I see the effect of this commit in arduino reference site ?

This is the sequence of events:
1. A member of the Arduino team responsible for documentation will review your pull request.
2. If they approve of your proposed change, they will merge your pull request. You will receive a notification from GitHub when that happens.
3. Some time after that, someone on the Arduino team responsible for the website will trigger an update of the [Arduino Language Reference section of the site](https://www.arduino.cc/reference/en). That update will be automatically generated from the content of the `master` branch of this repository and then published to the arduino.cc website.

So I can't give you any clear idea of a when, but at least you can get some idea of where in the sequence your changes are at.